adoc/release-5.0.0.adoc

```diff
@@ -64,3 +64,11 @@ You must update the {crio} configuration files on each node before upgrading, pl
 
 
 == Known Issues
+
+=== In the upgrade process, after the restart of crio and kubelet, some pods might not run properly
+
+This happens when there are multiple "NotReady" Pods. A workaround is to make sure that there are no PodSandboxs in "NotReady" state before the upgrade by draining the node before upgrading.
```

Comment:
I'd suggest something like this:

This can happen when there are multiple instances of a PodSandbox in a "NotReady" state. As a workaround please make sure to remove any pod in the "NotReady" state using `crictl rmp <podid>`. Further it is advisable to drain the node that is being upgrade before actually starting the upgrade procedure.

Reply:
I just put your text. Thanks.